include/grub/slr_table.h

```diff
@@ -119,6 +119,8 @@ struct grub_slr_entry_dl_info
   grub_uint64_t dl_handler;
   grub_uint64_t dce_base;
   grub_uint32_t dce_size;
+  grub_uint64_t dlme_base;
+  grub_uint32_t dlme_size;
   grub_uint64_t dlme_entry;
```

Comment:
(Forgot to add this comment to the review)

Currently, this field specifies the offset from DLME base, not the physical address of entry point. As such, it will never be bigger than `dlme_size`, so it may as well be changed to `grub_uint32_t`.

Reply:
https://github.com/TrenchBoot/grub/compare/4543040551b7163496e651e19572bcf662ba42eb..13bf69f550024b02fb94fa4073c84643f4d91574#diff-fa8593325ef57d162ad9d9472e2e6571ec4dba071acb7da6e07515bbe8658a63